### ISSUE TITLE
release csv label files for ssv2

### ISSUE THREAD

**Issue by nqx12348** [MCG-NJU/VideoMAE#27]
Hi, congratulate on your great work! 
Could you release label files (train.csv val.csv test.csv) for ssv2?
I tried to generate these files following the guidance in [DATASET.md](https://github.com/MCG-NJU/VideoMAE/blob/main/DATASET.md) and use them for evaluation on ssv2 by running [run_class_finetuning.py](https://github.com/MCG-NJU/VideoMAE/blob/main/run_class_finetuning.py) with --eval, but the accuracy is abnormal(too low), and I guess maybe the csv files I generated were wrong.
Could you release the csv files of K400 and ssv2 for everyone to download? Thanks!

**Comment by nqx12348** [comment]
Sorry I just found I used the pretrained model rather than finetuned model in evaluation😓
The finetuned model works well.

**Comment by klinic** [comment]
老哥，请问可以分享给我那些csv文件？我的邮件是3014457121@qq.com

**Comment by nqx12348** [comment]
> 老哥，请问可以分享给我那些csv文件？我的邮件是[3014457121@qq.com](mailto:3014457121@qq.com)

Hi, I have send you the csv files I made for ssv2, please check your email.

**Comment by greggg123** [comment]
@nqx12348 Would you be able to send the csv files to me too? Would greatly appreciate it! My email is achunla321@gmail.com

**Comment by klinic** [comment]
> @nqx12348 Would you be able to send the csv files to me too? Would greatly appreciate it! My email is [achunla321@gmail.com](mailto:achunla321@gmail.com)

off course

**Comment by greggg123** [comment]
@klinic Thank you so much! I also have been having trouble trying to run the pretraining itself in my terminal - how were you able to run pretraining using the given script file?

**Comment by klinic** [comment]
> @klinic Thank you so much! I also have been having trouble trying to run the pretraining itself in my terminal - how were you able to run pretraining using the given script file?

I've been preparing data sets for the past two days and haven't started running code yet. I have to ask the teacher whether to run this code this afternoon. The elder brother above ran successfully. You can @ him.
If I succeed in running the code, I will tell you at the first time.

**Comment by greggg123** [comment]
Thanks @klinic - @nqx12348 would you be able to give any help in terms of how to run the pretraining itself in the terminal?

**Comment by nqx12348** [comment]
> Thanks @klinic - @nqx12348 would you be able to give any help in terms of how to run the pretraining itself in the terminal?

Hi @greggg123 I didn't run the pretraining code since I use this codebase for fintuning on other datasets. I tried to run fintuning on ssv2 and activitynet following [FINETUNE.md](https://github.com/MCG-NJU/VideoMAE/blob/main/FINETUNE.md), and the code worked well for me. Follow the guidance in readme.md, prepare data/label/model files, and carefully check the arguments in the running script if you have difficulty running the script. Good luck!

**Comment by hxi667** [comment]
> 

Would you be able to send the csv files to me too? Would greatly appreciate it! My email is hxi@gs.zzu.edu.cn  .Thans

**Comment by hxi667** [comment]
@nqx12348 Would you be able to send the csv files to me too? Would greatly appreciate it! My email is [hxi@gs.zzu.edu.cn](mailto:hxi@gs.zzu.edu.cn) .Thans

**Comment by klinic** [comment]
> @nqx12348 Would you be able to send the csv files to me too? Would greatly appreciate it! My email is [hxi@gs.zzu.edu.cn](mailto:hxi@gs.zzu.edu.cn) .Thans

我刚刚发你了，你看看呢

**Comment by hxi667** [comment]
Thans,I get it




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月26日(星期五) 下午4:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [MCG-NJU/VideoMAE] release csv label files for ssv2 (Issue #27)





  
@nqx12348 Would you be able to send the csv files to me too? Would greatly appreciate it! My email is ***@***.*** .Thans
  
我刚刚发你了，你看看呢

**Comment by IemProg** [comment]
@nqx12348 Would you be able to send the csv files to me too? Would greatly appreciate it! My email is [imadeddine.ma@gmail.com](mailto:imadeddine.ma@gmail.com)

**Comment by yztongzhan** [comment]
Hi @IemProg and others! We hope to be of help to you by sharing our annotation files (train.csv, val.csv, test.csv) via this [link](https://drive.google.com/drive/folders/1cfA-SrPhDB9B8ZckPvnh8D5ysCjD-S_I?usp=share_link).
